Added tier 2 call page (#555)
RSE-Sheffield/RSE-Sheffield.github.io: pages/collaboration/tier2/tier2.md

Aside from the universities own HPC systems. The University of Sheffield is a partner organisation in two EPSRC Tier 2 HPC facilities; [JADE II](https://docs.hpc.shef.ac.uk/en/latest/other-uk-hpc-resources/jade2.html) and [Bede](https://docs.hpc.shef.ac.uk/en/latest/other-uk-hpc-resources/bede.html). As such we have a percentage share of the system available for University of Sheffield research. Both systems are large GPU compute clusters which have been designed to perform large scale computations in a broad range of domains. JADE II is specifically designed to support Machine Learning and Bede to support more general GPU computation spanning multiple compute nodes. All members of the university can access these facilities free of charge even if their work falls outside of EPSRCs domain.

> This call will specifically aim to enable researchers to use the Tier 2 HPC systems by providing funding for embedded support within research teams or with individual researchers. Embedded support will be provided by the Research Software Engineering group, a team of ~13 software engineering experts who work collaboratively across the entire university supporting [~40 active projects](https://rse.shef.ac.uk/collaboration/projects/) worth over £12M to the university. The call is keen to support researchers from a diverse research background, particularly those who are not traditional HPC users.

Author: ```suggestion
This call will specifically aim to enable researchers to use the Tier 2 HPC systems by providing funding for embedded support within research teams or with individual researchers. Embedded support will be provided by the Research Software Engineering group, a team of ~12 software engineering experts who work collaboratively across the entire university supporting [~40 active projects](https://rse.shef.ac.uk/collaboration/projects/) worth over £12M to the university. The call is keen to support researchers from a diverse research background, particularly those who are not traditional HPC users.
```

Applications should not be submitted for short pieces of consultancy work which can be completed in less than a day. Instead such requests should be submitted to the [Code Clinic](https://rse.shef.ac.uk/support/code-clinic/), a free consultancy service run by the RSE and Research IT teams. Alternatively for support requests you can raise a [ticket via ITS](https://shef.topdesk.net/solutions/forms/eeb223ec-fd1a-4436-b3fb-3c8414f9903b?token=5b2cdc96-fa45-48b7-b756-58c68884ebd2). Requests for embedded support beyond the 12 day limit will also not be considered. You should however discuss with the RSE team management about how to support your research [through grant funding](https://rse.shef.ac.uk/collaboration/) if a larger investment of specialist staff time is required.

If you are unsure about eligibility of scope then please contact [rse@sheffield.ac.uk](mailto:rse@sheffield.ac.uk) or book a code clinic session to discuss your project ideas.

> ## How Your Application will be Assessed (PR)

Author: ```suggestion
## How Your Application will be Assessed
```

---
title: Tier 2 Embedded Support
permalink: /collaboration/tier2/
slug: index
type: text
---

> # Open Call: Funding for Embedded support for High Performance Computing and Machine Learning

Reviewer: The font size of this is execssive.

![image](https://user-images.githubusercontent.com/628937/159484333-0969f219-a9d4-425d-bd90-a0f26b305b9b.png)

This is a general comment on the H1 styling, but the simplest fix is to switch to `##` and increment all the heading levels on the page.

Reviewer: Opened #558 related to this, to be resolved in the future

## Context

> High performance computing (HPC) is the processing of data and performing of computing calculations at high speeds. When surveyed (as part of the Sheffield Research Software survey 2020), roughly a quarter of researchers from all faculties at Sheffield expressed an interest in using HPC in the future. It may not be as difficult as you would think to do so! The barrier to entry for using specialist HPC facilities such as Graphics Processing Units (GPU) has been significantly lowered by the emergence of software to facilitate Machine Learning (ML) and AI. The purpose of this open call is to provide funding to support skilled Research Software Engineers (RSEs) to assist researchers in migrating their research workflows to suitable HPC systems. RSE support will be provided from the [Sheffield RSE group](https://rse.shef.ac.uk/). The RSE group is a team of 12 who provide collaborative embedded support for research across the whole of the university.

Reviewer: ```suggestion
High performance computing (HPC) is the processing of data and performing of computing calculations at high speeds. When surveyed (as part of the Sheffield Research Software survey 2020), roughly a quarter of researchers from all faculties at Sheffield expressed an interest in using HPC in the future. It may not be as difficult as you would think to do so! The barrier to entry for using specialist HPC facilities such as Graphics Processing Units (GPUs) has been significantly lowered by the emergence of software to facilitate Machine Learning (ML) and AI. The purpose of this open call is to provide funding to support skilled Research Software Engineers (RSEs) to assist researchers in migrating their research workflows to suitable HPC systems. RSE support will be provided from the [Sheffield RSE group](https://rse.shef.ac.uk/). The RSE group is a team of 12 who provide collaborative embedded support for research across the whole of the university.
```

Aside from the universities own HPC systems. The University of Sheffield is a partner organisation in two EPSRC Tier 2 HPC facilities; [JADE II](https://docs.hpc.shef.ac.uk/en/latest/other-uk-hpc-resources/jade2.html) and [Bede](https://docs.hpc.shef.ac.uk/en/latest/other-uk-hpc-resources/bede.html). As such we have a percentage share of the system available for University of Sheffield research. Both systems are large GPU compute clusters which have been designed to perform large scale computations in a broad range of domains. JADE II is specifically designed to support Machine Learning and Bede to support more general GPU computation spanning multiple compute nodes. All members of the university can access these facilities free of charge even if their work falls outside of EPSRCs domain.

> This call will specifically aim to enable researchers to use the Tier 2 HPC systems by providing funding for embedded support within research teams or with individual researchers. Embedded support will be provided by the Research Software Engineering group, a team of ~13 software engineering experts who work collaboratively across the entire university supporting [~40 active projects](https://rse.shef.ac.uk/collaboration/projects/) worth over £12M to the university. The call is keen to support researchers from a diverse research background, particularly those who are not traditional HPC users.

Reviewer: Previously stated it's a team of `12`, so is it `12` or `~13`?

Author: It literally changes mid sentence sometimes

## Example Case Study

**Project Title**: Migrating Lung Segmentation

**Application Theme**: Support for scaling up of Machine Learning/AI workflows

**Project Details**: We worked with Bilal Tahir’s group to migrate a lung segmentation workflow to JADE. The work required the use of Niftynet a Tensor Flow based open source platform for research in medical image analysis and image-guided therapy. Niftynet uses convolutional neural networks (CNNs) and was therefore good fit to GPU acceleration on JADE. The Niftynet software was not able to be installed on JADE using a conventional approach as the installation process requires an outgoing internet connection which is unavailable with JADE for security reasons. The RSE on this project helped create a singularity container (a form of software execution environment) which packages Niftynet and all required software for use on the JADE cluster. The end result was that sears within the team could run large scale medical image analysis workflows without having complex software installations to navigate.

> 

Reviewer 2: @frb-yousefi would you be able to help add in another case study related to your ML consultancy?

@mondus @ptheywood  Would it also be good to add a case study related to optimisation, GPU or otherwise? Do we have any examples?

Author: Need to get this out so later examples can follow

## Eligibility and What is on Offer

> This call is open to all research staff but with some conditions imposed by the Tier 2 facilities. Postdocs must have at least one year left on their contract (else should apply via their PI). Postdoctoral research students must have support from their supervisor.

Reviewer: PhD students are not `postdoctoral` surely?

```suggestion
This call is open to all research staff but with some conditions imposed by the Tier 2 facilities. Postdocs must have at least one year left on their contract (else should apply via their PI). Doctoral research students must have support from their supervisor.
```

Applications should not be submitted for short pieces of consultancy work which can be completed in less than a day. Instead such requests should be submitted to the [Code Clinic](https://rse.shef.ac.uk/support/code-clinic/), a free consultancy service run by the RSE and Research IT teams. Alternatively for support requests you can raise a [ticket via ITS](https://shef.topdesk.net/solutions/forms/eeb223ec-fd1a-4436-b3fb-3c8414f9903b?token=5b2cdc96-fa45-48b7-b756-58c68884ebd2). Requests for embedded support beyond the 12 day limit will also not be considered. You should however discuss with the RSE team management about how to support your research [through grant funding](https://rse.shef.ac.uk/collaboration/) if a larger investment of specialist staff time is required.

If you are unsure about eligibility of scope then please contact [rse@sheffield.ac.uk](mailto:rse@sheffield.ac.uk) or book a code clinic session to discuss your project ideas.

> ## How Your Application will be Assessed (PR)

Reviewer 2: Remove (PR)

* **Suitability of your application to make effective use of one of the Tier 2 systems (50%)**. This will include the potential for your research work to utilise the GPU equipment effectively. You do not have to identify which system you would like to target only which theme you are applying for. Criteria will be assessed by considering the software or data which you are using as part of your research.
* **Impact of the project on research work (25%)**. This is not a measure of the research quality but an assessment of the impact in which the embedded support will advance the research work. You should clarify how the activities of this project will advance your research.
* **Availability of resources and future ambitions (25%)**. You must identify researchers or members of your research team to work with the RSE over a period of time. It is vital that this activity is undertaken as a collaboration in order to get the maximum value from the embedded staff member. This criteria will also consider the current (and proposed) availability of software and data as well as availability of RSE team members to undertake the work.

## Application Process

> You should apply via the provided expression of interest google form below. This is an open call but applications will be reviewed in batches every 6 months. **The closing date for the next (first) round of reviews is Friday 22/04/2022 at 17:00. ** Applications received after this date will be reviewed in the next round of applications. Dates for future rounds will be announced on this page.

Reviewer: ```suggestion
You should apply via the provided expression of interest google form below. This is an open call but applications will be reviewed in batches every 6 months. **The closing date for the next (first) round of reviews is Friday 22/04/2022 at 17:00**. Applications received after this date will be reviewed in the next round of applications. Dates for future rounds will be announced on this page.
```

> [https://bit.ly/Tier2support](https://bit.ly/Tier2support) 

Reviewer: Url shortened links are generally not great, they can take you anywhere. 

It would be better to provide meaningful link text which connects directly to the form. 

```suggestion
[Application Form](https://docs.google.com/forms/d/e/1FAIpQLScsp1T57ohGnCA91D7KoKJPPtGiuVTY52u8S3q21I9VLLsH0Q/viewform) 
```

It would be even better if this were part of the above paragraph so the link made sense in context (not sure how to do a multiline suggestion, sorry)

> On receiving a completed application form we will contact you to arrange a short (less than 30m) online consultation to gather any additional technical information required to understand your project. The application form is purposefully short to ensure that we can discuss your requirements rather than expect you to provide all of these in the written application. 

Reviewer: ```suggestion
On receiving a completed application form we will contact you to arrange a short (less than 30 minutes) online consultation to gather any additional technical information required to understand your project. The application form is purposefully short to ensure that we can discuss your requirements rather than expect you to provide all of these in the written application. 
```

After review the projects will be ranked and estimates given as to when RSE time can be allocated. At the conclusion of your project a brief post-project report will be required.

> 

Reviewer 2: Remove all internal notes below

Author: Yes I just realised this.

> # Detail to Capture and Internal Notes

Reviewer: This section and below should not be here?

We should consider automating a google form to populate a github issue in a private repo. Reviewing can be done via a continuous cycle and we should have a Github project board with issues so that it is clear how they are prioritised and what is being worked on. 

## Google Form

## Designed to capture basic project details. Purposefully lightweight to encourage applications to progress to the discussion stage.

* **Applicant Name**
* **PI Name** if different from above
* Checkbox “If I am a postdoctoral researcher or postdoc with less than one year on my contract I have discussed and sought approval for this application with my line manager.”
* **Project Short Title**
* **Application theme**
* **Amount of funding requested**. RSE time is £386 per day. 
* **Project plan** (200 words): This should describe what the planned activities are and any details that will assist us in assessing how suitable your plan is for making effective use of the Tier 2 systems.

> * **Impact of the project ** (200 words): How will the embedded RSE effort change your current approach to research/experimentation.

Reviewer: ```suggestion
* **Impact of the project** (200 words): How will the embedded RSE effort change your current approach to research/experimentation.
```
